src/ImageSharp/Common/Helpers/SimdUtils.HwIntrinsics.cs

```diff
@@ -616,44 +616,6 @@ public static Vector256<float> MultiplyAdd(
                 return Fma.MultiplyAdd(vm1, vm0, va);
             }
 
-            if (Avx.IsSupported)
```

Comment:
nit: You can have a similar path for Arm64
```csharp
if (AdvSimd.IsSupported)
{
    return AdvSimd.FusedMultiplyAdd(va, vm1, vm0);
}
```

When you're able to move to .NET 9+, you can switch all this to just `Vector128.MultiplyAddEstimate(x, y, addend)` which handles this optimization for you.